src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -514,17 +514,21 @@ private static string GenerateUpgradeReport(string projectName, string backupPat
                         isPackageSourceMappingEnabled: isPackageSourceMappingEnabled);
 
                     var nuGetUI = uiService as NuGetUI;
-                    AddUiActionEngineTelemetryProperties(
-                        actionTelemetryEvent,
+
+                    actionTelemetryEvent["ActiveFilter"] = nuGetUI.PackageManagerControl.ActiveFilter;
```

Comment:
```suggestion
                    actionTelemetryEvent["ActiveTab"] = nuGetUI.PackageManagerControl.ActiveFilter;
```

filter makes it sound like it's the search query the customer typed in.

Since this is not in the NuGet Client SDK, I'd also love for the property to be renamed so it's clearer to people like me who don't spend a lot of time in the PM UI code. But that can be done in a separate PR.

Reply:
Ok, will rename